cirq/protocols/approximate_equality_protocol.py

```diff
@@ -94,6 +95,11 @@ def approx_eq(val: Any, other: Any, *, atol: Union[int, float] = 1e-8) -> bool:
     if isinstance(val, str):
         return val == other
 
+    if isinstance(val, sympy.core.mul.Mul) or isinstance(
+            other, sympy.core.mul.Mul):
+        delta = sympy.simplify(sympy.Abs(other - val))
+        return sympy.LessThan(delta, atol) == True
```

Comment:
Let's compare to `sympy.true` to make this a bit less confusing - it does work but it does take a bit of a sympy knowledge to understand what's happening here.

Reply:
So I know very little abot Sympy. Did you mean `sympy.LessThan(delta, atol) == sympy.true`? If so, I see an error:
`
E       assert not True
E        +  where True = <function approx_eq at 0x7f9a3f624ee0>(((t + 1.51) + s), ((t + 1.5) + s), atol=0.05)
E        +    where <function approx_eq at 0x7f9a3f624ee0> = cirq.approx_eq
`

I suspect it's because we need to convert to a plain Python True. Although I am not sure I understood you correctly and since I did not change the code, I'm not resolving this comment.